### PR TITLE
Improvement of the log

### PR DESCRIPTION
**Pull request by yuusuke**
I will demand an improvement in the log of the next two points.
The monitor is I want to know whether or not it so? Probe.
In previous log is monitor_0, interval 0 So it was found.
I want to now change it to a more easy-to-understand display

Please display the return code.
It is important in order for the user to analyze whether the finished from the return code that appeared in the log at any exit code resource agent.

### CODE REVIEW

File: include/crm/crm.h [hunk 1]
```diff
@@ -187,6 +187,7 @@ extern char *crm_system_name;
 #  define RSC_NOTIFIED	CRMD_ACTION_NOTIFIED
 
 #  define RSC_STATUS	CRMD_ACTION_STATUS
+#  define RSC_PROBE	CRMD_ACTION_STATUS"(probe)"
```
Comment (jnpkrn) [inline include/crm/crm.h]: would leave a space after `CRMD_ACTION_STATUS` as tokens are pasted together automatically by the preprocessor, regardless of space

Comment (yuusuke) [inline include/crm/crm.h]: Thank you for comment !
I did not know a lot about processing of preprocesser.
I intend to change it as follows.
```
#  define CRMD_ACTION_PROBE            "monitor(probe)"
```

**Comment by kgaillot** [comment]
The purpose of putting rc/status after CRM_XS is so they show up in the pacemaker detail log (pacemaker.log, corosync.log, or whatever is configured locally) and not the system log.

The goal is to put information of interest to system administrators in the system log, and anything more targeted to developers in the detail log. System administrators often complain that Pacemaker's logs are too confusing, so this is a relatively new feature we are using to make it easier for them.

**Comment by kgaillot** [comment]
I don't think it's appropriate to define a CRMD_ACTION_ constant for probe, since it's only a display value, and not something actually used. But I'd be open to defining an inline function for

    safe_str_eq(task, RSC_STATUS) && !interval? "probe" : task

**Comment by yuusuke** [comment]
Hi, Ken

The extra information, intended which has moved to the log of the developers was understanding.
However, rc when a failure occurs, I think that it is useful to understand what resource agent has finished in any exit code.
I divided the patch fixes. Please merge also this if you don't mind.

I accordance with the comments, we created an inline function that returns the action.
How does that look?

**Comment by kgaillot** [comment]
That makes sense, thanks! I'd prefer some minor changes:
* When printing rc, let's use %d (%s) for rc and its description (instead of %s rc=%d).
* Let's not print op->op_status in the system log, because it's not an exit code, and it has no meaning to someone who is not familiar with the code internals.
* Please close this request, and resubmit against the 1.1 branch, because I think this would be good to get into the 1.1.15 release.

**Comment by yuusuke** [comment]
Hi, Ken
Thank you for your comment
I fixed it now.
I close this request, we will create a new request for branch 1.1.

**Pull request closed without merging**